core/shared/utils/bh_vector.c

```diff
@@ -143,24 +143,26 @@ bh_vector_insert(Vector *vector, uint32 index, const void *elem_buf)
 {
     size_t i;
     uint8 *p;
+    bool ret = false;
 
     if (!vector || !elem_buf) {
         LOG_ERROR("Insert vector elem failed: vector or elem buf is NULL.\n");
-        return false;
+        goto RETURN;
```

Comment:
Seems we can return false directly here, no need to add a `RETURN` label?

Reply:
since `goto` is in the function, try using "one-exit-point"